rclcpp/src/rclcpp/utilities.cpp

```diff
@@ -224,10 +224,19 @@ rclcpp::remove_ros_arguments(int argc, char const * const argv[])
     &nonros_argv);
 
   if (RCL_RET_OK != ret) {
+    // Not using throw_from_rcl_error, because we may need to append deallocation failures.
+    exceptions::RCLErrorBase base_exec(ret, rcl_get_error_state());
```

Comment:
unless it's a convention that I haven't seen before, `exec` doesn't seem like an appropriate variable name (`base_exc` instead?)